hedera-node/src/main/java/com/hedera/services/fees/calculation/UsageBasedFeeCalculator.java

```diff
@@ -197,14 +194,17 @@ private FeeObject feeGiven(
 			FeeData prices,
 			ExchangeRate rate
 	) {
+		var sigUsage = getSigUsage(accessor, payerKey);
+		var usageEstimator = getTxnUsageEstimator(accessor);
 		try {
-			var sigUsage = getSigUsage(accessor, payerKey);
-			var usageEstimator = getTxnUsageEstimator(accessor);
-			var metrics = usageEstimator.usageGiven(accessor.getTxn(), sigUsage, view);
+			FeeData metrics = usageEstimator.usageGiven(accessor.getTxn(), sigUsage, view);
 			return FeeBuilder.getFeeObject(prices, metrics, rate);
-		} catch (Exception illegal) {
-			var msg = String.format("Unable to compute fee for %s, key %s!", accessor.getSignedTxn4Log(), payerKey);
-			throw new IllegalArgumentException(msg, illegal);
+		} catch (InvalidTxBodyException e) {
+			log.warn(
+					"Accessor argument {} malformed for implied estimator {}!",
+					accessor.getSignedTxn4Log(),
```

Comment:
The `Accessor argument` means the transaction body here. I think it will be more helpful to say the `The signed transaction body` to help people to understand what's wrong. 

But not a big deal nor worth the effort to make any changes now.